src/main/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextRBSFlight.java

```diff
@@ -9,11 +9,14 @@
 
 // NOTE: DO NOT USE. Currently just a shell class to exercise connection to Buffer Service.
```

Comment:
javadoc comment for the class? is this still correct?

Reply:
Thanks, fixed.